helpers/terms.inc.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+require_once(realpath(dirname(__FILE__)) .'/lookup.inc.php');
+
+function addTerm($name, $taxonomyId, $parentId, $sourceNote='', $useForTerms='', $scopeNotes='')
+{
+  // If term with this name already exists, don't add a new one.
+  $criteria = new Criteria;
+  $criteria->addJoin(QubitTerm::ID, QubitTermI18n::ID, Criteria::INNER_JOIN);
+  $criteria->add(QubitTerm::TAXONOMY_ID, $taxonomyId);
+  $criteria->add(QubitTermI18n::NAME, $name);
+
+  if (null === $term = QubitTerm::getOne($criteria))
+  {
+    // Create term
+    $term = new QubitTerm;
+    $term->taxonomyId = $taxonomyId;
+    $term->parentId = $parentId;
+    $term->name = $name;
+    $term->save();
+
+    // Deal with source note
+    if (!empty($sourceNote))
```

Comment:
Shouldn't `$sourceNote` be added to an existing term as well, like the `$useForTerms` and $scopeNotes?